specification/components/schemas/Meta.yaml

```diff
@@ -11,7 +11,7 @@ properties:
     description: |
       The level of security on the patients record, which affects which fields are populated on retrieval. The possible responses are:
       * U - unrestricted. All available data will be returned.
-      * R - restricted. Any sensitive data around the patients location, so `address`, `generalPractitioner` and `telecom`, will be removed from the response. Additionally, the death notification extension will be removed.
+      * R - restricted. Any sensitive data around the patients location, so `address`, `generalPractitioner` and `telecom`, will be removed from the response.
```

Comment:
Query about the use or Redacted and Removed. To me Redacted normally means that information is hidden so it can't be seen. To me Removed means taken away. The definition of Redacted seems to mean something slightly different (i.e. marked as Invalid). I'd like to understand more about whether these are meant to have the normal meaning or whether they are self-defining terms.